### DISCUSSION TITLE
TRMM Project

### DISCUSSION THREAD

**Discussion by needualas3script** [amidaware/tacticalrmm#1811] [General]
Hi Everyone, I'm new here, I'm a 2nd year student in Computing in Networking Technologies. I have been tasked to do a project on TRMM and I have until 24th of May to submit and defend it. I will be doing the traditional installation. I need someone with experience to put me through each steps to complete this project. I would really appreciate every help render to me during the course of my project. I would be looking forward to hear from anyone who will tutor and mentor me on this journey.

Thanks.

**Comment by dinger1986** [comment]
hello! welcome :), all the steps are here in our docs https://docs.tacticalrmm.com/install_server/, get a real domain its easier, anything specific you are stuck with please come back and ask

**Reply by needualas3script** [reply]
Hi @dinger1986, I have started working on creating the TRMM through traditional installation, I bought a domain from cloudfare.com but when I get to the stage to put my wild certificate I got confused and don't know how to go about it, so I am stucked at the moment. Can you put me through on that and also do you know how I can create a snapshot on mobaxterm ssh terminal.
 Thank you and looking forward to hearing from you.

**Reply by dinger1986** [reply]
Just need to add the dns records.

https://community.cloudflare.com/t/how-to-set-up-dns-records-for-cloudflare/586151

I don't use mobaxterm

**Reply by needualas3script** [reply]
Already have your own SSL certificate? Call the install script with the --use-own-cert flag, like this:


./install.sh --use-own-cert
Ensure your cert and private key exist on the server as the install script will prompt you for the locations of these 2 files.
Also ensure that it is a legitimate, trusted certificate and includes the full chain for proper validation. Do not use this option with self-signed certs.

I have already added the 3 dns recods rmm mesh and api. I just need to know how to locate the ssl certificate on the server.

**Reply by dinger1986** [reply]
do you have your certificate? where did you get it?

**Reply by needualas3script** [reply]
I created an SSL certificate on cloudflare where I registered my domain from

**Reply by dinger1986** [reply]
does that not then require using a cloudflare proxy? why not just use the wildcard generation using letsencrypt?

**Reply by needualas3script** [reply]
I would prefer the wildcard generation key that's what I need to know how to do, but after I ran the install script yesterday it gave me an error message.

**Reply by dinger1986** [reply]
so when it asks you need to create the acme verification txt record with the long sting, verify it with google dig and then hit enter on the terminal to continue the install

**Comment by needualas3script** [comment]
it gave me the value and also the TXT record name as _acme-challenge.emeraldpark.com do I have to add the domain name or just put only the _acme-challenge in the DNS record field

**Reply by dinger1986** [reply]
![image](https://github.com/amidaware/tacticalrmm/assets/7244447/5cd7f899-d868-437c-9191-1219556c95a5)
If you try it its fairly obvious as per the picture above, it adds the full domain etc before the full domain and has added extra dots etc, so you need to put  _acme-challenge in the Name field and the string into the Content field, then verify

**Reply by needualas3script** [reply]
Thank you so much for your time, that worked.
